ldms/src/ldmsd/ldmsd_config.c

```diff
@@ -316,8 +316,8 @@ int ldmsd_load_plugin(char* inst_name, char *plugin_name, char *errstr, size_t e
 	struct ldmsd_plugin *api = load_plugin(plugin_name);
```

Comment:
Love the switch to "plugin=" instead of "instance="!

But this "api = load_plugin(plugin_name);" now needs to be moved down after the "if (!plugin_name)" and before the "switch (api->type)" line.

Reply:
Does this match what you were looking for? I liked you suggestion and think it makes sense. I agree that instance is way overloaded. I have a patch that will be up shortly that addresses that issue.

Reply:
Yes, that change to plugin= instead of instance= is what I was looking for! (And I implmented over in PR #1600). You can get the rest of the little changes for just this plugin= stuff in commit da2c868e6ad8a4bdf93c05fdb88dbcff1080495a.